core/amber/src/main/scala/edu/uci/ics/texera/web/service/ExecutionRuntimeService.scala

```diff
@@ -74,7 +79,15 @@ class ExecutionRuntimeService(
 
   // Receive Interaction
   addSubscription(wsInput.subscribe((req: WorkflowInteractionRequest, uidOpt) => {
-    client.sendAsync(RetrieveWorkflowState())
+    if (logConf.isEmpty) {
+      logger.info(
+        "Fault tolerance log folder is not established. Unable to take a global checkpoint."
+      )
```

Comment:
it should be an error log and raise an exception?

Reply:
changed it to an assertion